pyproject.toml

```diff
@@ -1,21 +1,28 @@
-[tool.commitizen]
-version = "4.1.1"
-tag_format = "v$version"
-version_files = [
-    "pyproject.toml:version",
-    "commitizen/__version__.py",
-    ".pre-commit-config.yaml:rev:.+Commitizen",
-]
-
-[tool.poetry]
+[project]
 name = "commitizen"
 version = "4.1.1"
 description = "Python commitizen client tool"
-authors = ["Santiago Fraire <santiwilly@gmail.com>"]
-license = "MIT"
-keywords = ["commitizen", "conventional", "commits", "git"]
+authors = [{ name = "Santiago Fraire", email = "santiwilly@gmail.com" }]
```

Comment:
wanna add `maintainers` section?
https://packaging.python.org/en/latest/specifications/pyproject-toml/#authors-maintainers

Reply:
Yep, why not haha